content/integrations/streaming/datadog.textile

```diff
@@ -0,0 +1,57 @@
+---
+title: Datadog
+meta_description: "Connect Ably and Datadog to monitor messages, channels, and connections in realtime, integrating your Ably statistics with your existing Datadog setup."
+meta_keywords: "Datadog, integrations, statistics, metrics, monitoring, analytics, enterprise"
+---
+
+The Ably "Datadog":https://docs.datadoghq.com/integrations/ably/ integration enables you to monitor your application's statistics. Every 60 seconds, Ably streams a comprehensive set of "statistics":/docs/metadata-stats/stats#metrics to the Datadog API.
+
+h2(#setup). Setup the Datadog integration
+
+To connect Ably with Datadog, you must "request":https://ably.com/support access to the integration. Once granted, you can authorize the integration through Datadog's "OAuth":https://docs.datadoghq.com/developers/integrations/oauth_for_integrations/ flow. This process requires the @api_keys_write@ scope, allowing Ably to push data to your Datadog account.
+
+Once the integration is active, Datadog provides a specific Ably "dashboard":https://docs.datadoghq.com/integrations/ably/, enabling you to monitor key metrics without extra setup.
+
+The following steps setup the Datadog integration:
+
+# In Datadog, go to *Integrations*, find the *Ably* tile, and click *Install Integration*.
+# Click *Connect Accounts* to start the authorization process. Datadog redirects you to Ably.
+# Log in to your *Ably* account.
+# Select your application from the *Your Apps* page.
+# Navigate to *Integrations*, and select *Connect to Datadog*.
+# Datadog authorization page, authorize Ably using *OAuth* to grant access. The required authorization scope is: @api_keys_write@.
+# After completing authorization, you will be redirected to the *Ably dashboard*, and the process is complete.
+
+h2(#remove). Remove access
+
+Removing access disconnects Ably from Datadog, stopping data transmission and revoking authorization. Follow the steps remove the Ably and Datadog integration using either platform.
+
+h3(#in-ably). Remove access using Ably
+
+* Open your application's integration settings.
+* Click *Remove* next to the Datadog integration.
+* Ably revokes OAuth credentials and stops metric transmission.
+
+h3(#in-datadog). Remove access using Datadog
+
+* Remove associated Ably API keys via *Integrations* or *API Keys*.
+* Adjust scopes or entirely revoke OAuth tokens if necessary.
+
+h2(#lite). Datadog lite
+
+Datadog Lite is a lightweight version of the full Datadog integration that sends a reduced set of "statistics":/docs/metadata-stats/stats#metrics to the Datadog API, available for "enterprise":/docs/pricing/enterprise customers. This integration is designed for use cases where full statistics are not required, such as when you only need to monitor a limited number channels or connections.
+
+The following statistics are streamed from Ably to Datadog:
```

Comment:
```suggestion
The following statistics are streamed from Ably to Datadog using the Lite integration:
```

Reply:
The lite mertrics are those that match the following regexps:
```
/^messages\.all\.all\..*$/,
/^messages\.all\.messages\..*$/,
/^messages\.all\.presence\..*$/,
/^messages\.inbound\.all\.all\..*$/,
/^messages\.outbound\.all\.all\..*$/,
/^connections\.all\.peak$/,
/^channels\.peak$/,
/^push\.channelMessages$/,
/^messages\.persisted\.messages\..*$/,
/^messages\.persisted\.presence\..*$/,
```

Reply:
So are a longer list that what's shown below rather than the root of each metric name, which will include `count`, `data`, `uncompressed` for example.